install.sh

```diff
@@ -4,23 +4,25 @@
 
 set -e
 
-case $(uname -s) in
+case "$(uname -s)" in
```

Comment:
This change does not prevent word splitting. Please revert.

Reply:
Done.